### ISSUE TITLE
Support for Sonatype Portal Publisher API

### ISSUE THREAD

**Issue by d10xa** [sbt/sbt-ci-release#333]
I’m encountering a 404: Not Found error when using the sbt-ci-release plugin after switching to the new Sonatype Portal Publisher API. The issue seems to be related to the URL used by the plugin, which points to the legacy API instead of the new one.

error:

```
2024-11-17 17:46:40.288Z error [Sonatype] 
wvlet.airframe.http.HttpClientException: [404: Not Found] Request failed: <!DOCTYPE html>...
    at xerial.sbt.sonatype.SonatypeClient.stagingRepositoryProfiles(SonatypeClient.scala:82)
    at xerial.sbt.sonatype.SonatypeService.stagingRepositoryProfiles(SonatypeService.scala:116)
    ...
```

Full logs: [GitHub Actions logs](https://github.com/d10xa/json-log-viewer/actions/runs/11880975242/job/33104765369)

Relevant Configuration:

	•	Plugin version: sbt-ci-release 3.11.2
	•	sonatypeCredentialHost is set to sonatypeCentralHost.

Could you please provide guidance on how to configure the sbt-ci-release plugin to work with the new Sonatype Portal Publisher API? Specifically, how should the publishTo and sonatypeCredentialHost settings be adjusted for the updated API endpoint (https://central.sonatype.com/api/v1/publish)? Any additional examples or documentation would be greatly appreciated.

**Comment by Andrapyre** [comment]
@d10xa , your version of `sbt-ci-release` is too old [here](https://github.com/d10xa/json-log-viewer/blob/main/project/plugins.sbt#L7). Working support for publishing to Sonatype Central only shipped with `1.7`. Here's an example of one of the first repos that we got working: https://github.com/PerformanceIMMO/pg-event-store/blob/main/build.sbt

**Comment by d10xa** [comment]
Thank you, @Andrapyre! This example solved my problem.
